v1/system.go

```diff
@@ -0,0 +1,41 @@
+package v1
+
+import (
+	"context"
+
+	client "github.com/hashicorp/nomad-openapi/clients/go/v1"
+)
+
+type System struct {
+	client *Client
+}
+
+func (c *Client) System() *System {
+	return &System{client: c}
+}
+
+func (s *System) SystemApi() *client.SystemApiService {
+	return s.client.apiClient.SystemApi
+}
+
+func (s *System) GarbageCollection(ctx context.Context) (interface{}, error) {
```

Comment:
```suggestion
func (s *System) GarbageCollect(ctx context.Context) (interface{}, error) {
```

Reply:
Should this just return err?  What is this result?

Reply:
I had tried to get it to just return err, but I couldn't get the test to run with an error.  So I ended up attaching the HTTP response as a result and the test just checks to see that it's not nil.

You're absolutely right, the result has no bearing on the  test, I just couldn't get the test to compile/run.